packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -116,29 +147,32 @@ class OpPerfTelemetry {
 					msg.type === MessageType.Operation &&
 					this.clientSequenceNumberForLatencyStatistics === msg.clientSequenceNumber
 				) {
+					// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
The `if` check right above this needs to consider `isSamplingDisabled` like the other ones we updated, so we make sure in that case all ops have these bits updated in their `latencyStats`.

Reply:
Updated in commit # c53a97